packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationFileUploadAdapter.ts

```diff
@@ -189,18 +189,20 @@ export class AzureCommunicationFileUploadAdapter implements FileUploadAdapter {
  * @param fileUploadUiState {@link FileUploadsUiState}
  * @private
  */
-export const convertFileUploadsUiStateToMessageMetadata = (fileUploads?: FileUploadsUiState): FileSharingMetadata => {
-  const fileMetadata: AttachmentMetadata[] = [];
+export const convertFileUploadsUiStateToMessageMetadata = (
+  fileUploads?: FileUploadsUiState
+): FileSharingMetadata | undefined => {
   if (fileUploads) {
+    const fileMetadata: AttachmentMetadata[] = [];
     Object.keys(fileUploads).forEach((key) => {
       const file = fileUploads[key];
       if (!file.error && file.metadata) {
         fileMetadata.push(file.metadata);
       }
     });
+    return { fileSharingMetadata: JSON.stringify(fileMetadata) };
```

Comment:
We probably want to check length here too.
In case if there is only 1 item and it has an error, an empty array value will be used for `fileMetadata`